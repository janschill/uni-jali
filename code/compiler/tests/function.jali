func f x =
  x + 2
end

f