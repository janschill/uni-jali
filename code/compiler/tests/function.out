Closure("f",["x"],Prim("+",Variable"x",Constant(IntegerValue2)),[])